docs/source/55_api.md

```diff
@@ -339,6 +339,52 @@ val bcastFeatures = sc.loadFeatures("my/features.adam").broadcast()
 val readsByFeature = reads.broadcastRegionJoinAgainst(bcastFeatures)
 ```
 
+#### Examples of real-world analyses possible with the RegionJoin API
```

Comment:
I would drop the "Examples of..." header and segue in with a paragraph before saying "To demonstrate how these APIs can be used, we'll walk through three common queries that can be written using the region join. They are X, Y, and Z." The subheaders should stay.